modules/navier_stokes/src/fvkernels/NSFVInterfaceTransfer.C

```diff
@@ -0,0 +1,39 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "NSFVInterfaceTransfer.h"
+#include "NS.h"
+
+registerMooseObject("NavierStokesApp", NSFVInterfaceTransfer);
+
+InputParameters
+NSFVInterfaceTransfer::validParams()
```

Comment:
there s already an ambient convection kernel that does roughly this

can you use that?

Reply:
if you cant, then interfacetransfer is not a good name. It s not precise enough

Maybe NSFVMixturePhaseInterface

Reply:
Thanks. Changed to `NSFVMixturePhaseInterface`. I don't want to use ambient convection because the name is very misleading regarding what this kernel is supposed to do (interfacial mass transfer). This model will later couple with specific mass transfer models for molten salts.